docs/dev/topology-over-raft.md

```diff
@@ -164,6 +164,8 @@ When tablet is not in transition, the following invariants hold:
 1. The storage layer (database) on any node contains writes for keys which belong to the tablet only if
     that shard is one of the current tablet replicas.
 
+When tablet table changes there is an event notification sent (cql_transport::event::event_type::TABLET_CHANGE). Currently it only informs that something changed without specifying details. The body of the message is empty (except the event type).
+
```

Comment:
This needs to be documented in docs/dev/protocol-extensions.md.

We need to think about providing change information. The tablets table is liable to grow to 100k rows, we don't want all clients to reread 100k rows on every migration.

We might encode a list of rows with each event, each row corresponding to a row in the tablets table, and add special events for merge/split.

Reply:
I moved this addition to documentation to `docs/dev/protocol-extensions.md`.